adaptive_breakpoints/example/lib/main.dart

```diff
@@ -112,10 +110,10 @@ class _AdaptiveContainer extends StatelessWidget {
   /// container will expand to fill all available space in its parent, unless
   /// the parent provides unbounded constraints, in which case the container
   /// will attempt to be as small as possible.
-  final Widget child;
+  final Widget? child;
```

Comment:
should child be nullable? I don't know what the standard is, or what even happens when child is null.

Reply:
From the documentation in this file, `child` is intended to be nullable. It is only used as the `child` of a `Container`, which is also nullable.

`child` has never taken the value of null in this package, but it might do so in the future, so I think it's best to keep it nullable.